Description: Upstream changes introduced in version 0.8.1-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 yojson (0.8.1-1) unstable; urgency=low
 .
   * Initial release. (Closes: #605681)
 .
 The person named in the Author field signed this changelog entry.
Author: Sylvain Le Gall <gildor@debian.org>
Bug-Debian: http://bugs.debian.org/605681

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- yojson-0.8.1.orig/Makefile
+++ yojson-0.8.1/Makefile
@@ -7,7 +7,7 @@ PACKS = easy-format,biniou
 
 .PHONY: default all opt install doc
 default: META all opt
-all: yojson.cmo
+all: yojson.cmo 
 opt: yojson.cmx ydump
 
 ifndef PREFIX
@@ -60,6 +60,10 @@ ydump: yojson.cmx ydump.ml
 	ocamlfind ocamlopt -o ydump -package $(PACKS) -linkpkg \
 		yojson.cmx ydump.ml
 
+ydump.byte: yojson.cmo ydump.ml
+	ocamlfind ocamlc -o ydump -package $(PACKS) -linkpkg \
+		yojson.cmo ydump.ml
+
 doc: doc/index.html
 doc/index.html: yojson.mli
 	mkdir -p doc
